docs/docs.html

```diff
@@ -136,13 +136,15 @@ <h5 id="remote-files">Parse remote file</h5>
 					<div class="grid-50">
 						<pre><code class="language-javascript">Papa.parse(url, {
 	download: true,
+	downloadTimeout: 5000,
```

Comment:
XMLHttpRequest calls it "timeout". I prefer to use the same name.

Reply:
I think the naming is correct. Addding the unit will only makes it longer with no extra information

Reply:
I believe it's incorrect to say that it doesn't add any information: it tells you exactly how to use the API at the call site instead of forcing the consumer to look up documentation or, worse, the source code.

As an example, what do you think the following API does?:
```javascript
Cache.prototype.setEvictionTime = function(timeout) { ... } ;
// what unit is timeout?

// vs

Cache.prototype.setEvictionTime = function (timeoutMs) { ... };
// Alright! I know what the parameter is 🥳
```

Reply:
There is more to it. The config value is passed to parseInt (I tried to use radix 10 but the tests fail). The [parseInt](https://www.w3schools.com/jsref/jsref_parseint.asp) method also parses octal and hexadecimal values. So as things are now calling it timeoutMs is not correct in my opinion.

Reply:
@demetris-manikas how does the number-base affect what unit it is?
2ms = 2 milliseconds
0x10ms = 2 milliseconds, also.